TeslaMateAgile/Data/Options/EnerginetOptions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace TeslaMateAgile.Data.Options;
+
+public class EnerginetOptions
+{
+    [Required]
+    public string BaseUrl { get; set; }
+
+    [Required]
+    public EnerginetRegion Region { get; set; }
+
+    [Required]
```

Comment:
While this field is required, the validation won't work here as it will just default to zero, we should add a `[Range(min, max)]` attribute here additionally with sensible values, maybe 1 - 2?

Reply:
Not sure exactly what you mean here?

Reply:
The VAT field defaults to zero and the `[Required]` attribute allows it to be zero, so it lets you get away with not configuring it at all. In my testing I thought it wasn't working because everything was coming out as 0 but that ended up being because the VAT was set to 0 and it was multiplying all the prices by 0.

The fix for this would be to add an additional attribute under the `[Required]` one to show the valid range of numbers for the VAT field, which is `[Range(min, max)]` e.g. `[Range(1, 2)]` which would allow VAT rates of 100% to 200% for example, not sure what is a sensible range to allow.